tests/integration/test_upgrade_charm.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+import json
+import logging
+import urllib.request
+from pathlib import Path
+
+import pytest
+import yaml
+from helpers import get_unit_address  # type: ignore[import]
+
+log = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+# app_name = "am"
+app_name = METADATA["name"]
```

Comment:
app_name is optional in the args below; it will be loki-k8s by default.
Here you can provide a custom name
```suggestion
app_name = "loki"
```